src/mechanics/integrationtypes/IntegrationType1D2NLobatto.cpp

```diff
@@ -0,0 +1,102 @@
+#ifdef ENABLE_SERIALIZATION
+#include <boost/archive/binary_oarchive.hpp>
+#include <boost/archive/binary_iarchive.hpp>
+#include <boost/archive/xml_oarchive.hpp>
+#include <boost/archive/xml_iarchive.hpp>
+#include <boost/archive/text_oarchive.hpp>
+#include <boost/archive/text_iarchive.hpp>
+#endif //ENABLE_SERIALIZATION
+
+#ifdef ENABLE_VISUALIZE
+#include "visualize/VisualizeEnum.h"
+#endif // ENABLE_VISUALIZE
+
+#include "mechanics/integrationtypes/IntegrationType1D2NLobatto.h"
+#include "mechanics/elements/ElementShapeFunctions.h"
+#include "boost/math/special_functions/legendre.hpp"
+#include "math/Polynomial.h"
+
+// constructor
+NuTo::IntegrationType1D2NLobatto::IntegrationType1D2NLobatto(int numIps)
```

Comment:
Use direct initialization for `nIps`, i.e `IntegrationType1D2NLobatto(int numIps) : nIps(numIps) { ... }` and then just use `nIps` throughout the constructor.